Gemfile.lock

```diff
@@ -266,125 +268,127 @@ GEM
+    airbrake-ruby (5.2.1)
       rbtree3 (~> 0.5)
     ansible (0.2.2)
     ar_multi_threaded_transactional_tests (0.5.0)
       activerecord (>= 4.2.0, < 6.2.0)
-    ast (2.4.1)
+    ast (2.4.2)
     attr_encrypted (3.1.0)
       encryptor (~> 3.0.0)
-    audited (5.4.2)
+    audited (5.4.3)
       activerecord (>= 5.0, < 7.2)
       request_store (~> 1.2)
-    autoprefixer-rails (9.4.8)
+    autoprefixer-rails (9.8.6.5)
       execjs
-    awesome_print (1.6.1)
-    aws-eventstream (1.2.0)
-    aws-partitions (1.329.0)
-    aws-sdk-core (3.100.0)
-      aws-eventstream (~> 1, >= 1.0.2)
-      aws-partitions (~> 1, >= 1.239.0)
+    awesome_print (1.9.2)
+    aws-eventstream (1.3.0)
+    aws-partitions (1.887.0)
+    aws-sdk-core (3.191.0)
+      aws-eventstream (~> 1, >= 1.3.0)
+      aws-partitions (~> 1, >= 1.651.0)
+      aws-sigv4 (~> 1.8)
+      jmespath (~> 1, >= 1.6.1)
+    aws-sdk-ecr (1.69.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-      jmespath (~> 1.0)
-    aws-sdk-ecr (1.3.0)
-      aws-sdk-core (~> 3)
-      aws-sigv4 (~> 1.0)
-    aws-sdk-kms (1.23.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-kms (1.77.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.45.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-s3 (1.143.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sdk-kms (~> 1)
-      aws-sigv4 (~> 1.1)
-    aws-sigv4 (1.6.1)
+      aws-sigv4 (~> 1.8)
+    aws-sigv4 (1.8.0)
       aws-eventstream (~> 1, >= 1.0.2)
-    base64 (0.1.1)
+    base64 (0.2.0)
+    bigdecimal (3.1.6)
     binding_of_caller (0.8.0)
       debug_inspector (>= 0.0.1)
-    bootsnap (1.4.5)
-      msgpack (~> 1.0)
+    bootsnap (1.18.3)
+      msgpack (~> 1.2)
     bootstrap-sass (3.4.1)
       autoprefixer-rails (>= 5.2.1)
       sassc (>= 2.0.0)
     bootstrap3-datetimepicker-rails (4.17.47)
       momentjs-rails (>= 2.8.1)
     brakeman (5.4.1)
     builder (3.2.4)
-    bundler-audit (0.9.0.1)
+    bundler-audit (0.9.1)
       bundler (>= 1.2.0, < 3)
       thor (~> 1.0)
     byebug (8.2.5)
-    chef-utils (18.1.29)
+    chef-utils (18.4.2)
       concurrent-ruby
-    coderay (1.1.1)
+    coderay (1.1.3)
     commonmarker (0.23.10)
-    concurrent-ruby (1.2.2)
-    connection_pool (2.2.1)
-    crack (0.4.3)
-      safe_yaml (~> 1.0.0)
+    concurrent-ruby (1.2.3)
+    connection_pool (2.4.1)
+    crack (0.4.6)
+      bigdecimal
+      rexml
     crass (1.0.6)
-    dalli (3.2.3)
+    dalli (3.2.7)
+      base64
     ddtrace (0.45.0)
       msgpack
     debug_inspector (0.0.3)
     diffy (3.4.2)
-    dogstatsd-ruby (3.0.0)
-    domain_name (0.5.20190701)
-      unf (>= 0.0.5, < 1.0.0)
-    doorkeeper (5.6.6)
+    dogstatsd-ruby (3.3.0)
+    domain_name (0.6.20240107)
+    doorkeeper (5.6.8)
       railties (>= 5)
-    dotenv (2.2.1)
+    dotenv (2.8.1)
     encryptor (3.0.0)
     erubi (1.12.0)
-    erubis (2.7.0)
-    execjs (2.7.0)
-    faraday (2.7.11)
+    execjs (2.9.1)
```

Comment:
### [execjs](https://github.com/rails/execjs)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.7.0 → 2.9.1

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/execjs/2.7.0/2.9.1)_]





<details>
<summary>Commits</summary>

A change of **85** commits. See the full changes on [the compare page](https://github.com/rails/execjs/compare/v2.7.0...v2.9.1).

These are the first 10 commits:
- (af43a57) [Undefine global `process`](https://github.com/rails/execjs/commit/af43a575f81e38f3d3b53935036423de29989803)
- (b0be19c) [Remove process and other node globals from `this`](https://github.com/rails/execjs/commit/b0be19c73ded5f20c2c86c8e546558e09da4e2a5)
- (5c6f4a9) [Normalize `console` in JSC](https://github.com/rails/execjs/commit/5c6f4a9115ebc86a9607a32ac7d15f9b7e5eb47c)
- (e57706f) [Merge pull request #43 from josh/undef-process](https://github.com/rails/execjs/commit/e57706ff068771e6ef9b5489b248ef5e77e4106e)
- (73bdb09) [Add note about installing and testing mini_racer](https://github.com/rails/execjs/commit/73bdb09a9b01e9f021c41eb1e5fe1d117c38b78a)
- (ff3f0fd) [Merge pull request #48 from junaruga/feature/contributing-mi…](https://github.com/rails/execjs/commit/ff3f0fd993653d3fd742fe2174a8b23baae3cc87)
- (35b612c) [Show Node version if EXECJS_RUNTIME is Node](https://github.com/rails/execjs/commit/35b612c85d3a8ff48a902b879b24d683f4177434)
- (cd417bc) [Update README.md](https://github.com/rails/execjs/commit/cd417bc3ecca16b40afe41fc553403283afec055)
- (5260f2d) [Update Ruby versions in .travis.yml](https://github.com/rails/execjs/commit/5260f2d6f57505e18494c9ec6f8fb6d15c1b0c24)
- (76363cf) [Merge pull request #61 from tricknotes/update-ruby-versions](https://github.com/rails/execjs/commit/76363cf1195ed59260a8a484522ad991b069a554)


</details>